server/src/main/java/org/apache/druid/server/coordinator/CoordinatorDynamicConfig.java

```diff
@@ -48,21 +49,21 @@
 {
   public static final String CONFIG_KEY = "coordinator.config";
 
-  private final long millisToWaitBeforeDeleting;
+  private final long millisLagSinceCoordinatorBecomesLeaderBeforeCanMarkAsUnusedOvershadowedSegments;
```

Comment:
Do you think it's easier for everyone, for example, people who are not Druid developers themselves?

Reply:
I also think that name is too long, suggest `dropOvershadowedSegmentsWaitMillis` instead

Reply:
Or `markOvershadowedAsUnusedInitialWaitMillis`

Reply:
I don't like the loss of meaning or even confusion in the "initial" adjective. Also, other configuration parameter names in this class don't drop the "Segments" part, except for `replicantLifetime` and `replicationThrottleLimit` which are bad names anyway. I also wanted to keep the "MarkAsUnused" part together, because it appears in different places in the code. What about `leadingTimeMillisBeforeCanMarkAsUnusedOvershadowedSegments`?